api/include/opentelemetry/trace/span.h

```diff
@@ -167,5 +169,14 @@ class Span
   // AddEvent).
   virtual bool IsRecording() const noexcept = 0;
 };
+
```

Comment:
Same here: this is internal detail. You probably don't need to introduce it in your commit.